tests/installation/start_install.pm

```diff
@@ -108,10 +108,10 @@ sub run {
         && !get_var("UPGRADE")
         && !check_var('VIDEOMODE', 'text'))
     {
-        my $counter = 10;
+        my $counter = 20;
         while ($counter--) {
             send_key $cmd{instdetails};
-            last if check_screen 'installation-details-view', 5;
+            last if check_screen 'installation-details-view', 20;
```

Comment:
Won't it be better to just increase number of iterations? As now we will wait for 20 seconds before trying to send key again. Otherwise LGTM.